apps/box/src/api/bloxHardware.ts

```diff
@@ -0,0 +1,37 @@
+export interface Blox {
```

Comment:
Should we maybe give functionland these schemes before delivery, just to give them a headstart on the api scheme we are using?

Reply:
talked offline about this